src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -458,10 +458,10 @@ export class Workbench implements IPartService {
 
 		// Backup File Service
 		const workspace = this.contextService.getWorkspace();
-		serviceCollection.set(IBackupFileService, this.instantiationService.createInstance(BackupFileService, workspace ? workspace.resource : null));
+		serviceCollection.set(IBackupFileService, this.instantiationService.createInstance(BackupFileService, this.windowService.getCurrentWindowId()));
```

Comment:
It could, and it did prior to https://github.com/Microsoft/vscode/pull/16303/commits/85bf25fd518130aca11be7ec268950b12e3b4fa0. Aren't less deps generally a good thing?